packages/@sfx/sayt-plugin/test/unit/common/index.test.ts

```diff
@@ -1,20 +1,44 @@
-import { expect, stub } from '../../utils';
-import * as Sample from '../../../src';
-
-describe('Test Import Common', () => {
-  let testData: Sample.DummyInterface = {
-    label: 'test',
-    number: 6,
-  };
-  it('should return the label string capitalized', () => {
-    expect(Sample.testFunc(testData)).to.equal('TEST');
+import { expect, sinon, stub } from '../../utils';
+import SaytPlugin from '../../../src/sayt-plugin';
+import * as SaytPackage from 'sayt';
+
+describe('SaytPlugin', () => {
+  let saytPlugin: any;
+
+  beforeEach(() => {
+    saytPlugin = new SaytPlugin();
+  });
+
+  describe('metadata getter', () => {
+    it('should not specify any dependencies', () => {
+      expect(saytPlugin.metadata.depends).to.deep.equal([]);
+    });
+
+    it('should have the name `sayt`', () => {
```

Comment:
I would put this first for consistency because it appears first in the metadata object.